### PR TITLE
Addresses Issue 14

### PR DESCRIPTION
**Pull request by vpanchal-supra**
This PR includes breaking changes, and those changes are mentioned below:
- Converted `getSendTxPayload` into a `sync` method from `async`
- Changed parameters of below given methods,
  - `sendTxUsingSerializedRawTransaction`, `createRawTxObject`, `transferSupraCoin`, `transferCoin`, `publishPackage`   
- `createRawTxObject` and `createSerializedRawTxObject` methods are no longer a `static` method

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -988,52 +1050,35 @@ export class SupraClient {
...
-    let maxGas = BigInt(50000);
-    if (
-      BigInt(0) + maxGas * BigInt(100) >
-      (await this.getAccountSupraCoinBalance(senderAccount.address()))
-    ) {
-      throw new Error("Insufficient Supra Coins");
-    }
-    if (
-      BigInt(amount) >
-      (await this.getAccountCoinBalance(senderAccount.address(), coinType))
```
Comment (isaacdoidge) [inline src/index.ts:1030]: Why have these checks been removed?

File: src/constants.ts [hunk 2]
```diff
@@ -2,11 +2,12 @@ export const DEFAULT_CHAIN_ID = 6;
...
-export const DEFAULT_GAS_UNIT_PRICE = BigInt(100);
 export const DEFAULT_MAX_GAS_UNITS = BigInt(500000);
 export const DEFAULT_TX_EXPIRATION_DURATION = 300; // 5 Minutes
 export const MILLISECONDS_PER_SECOND = 1000;
 export const SUPRA_FRAMEWORK_ADDRESS =
   "0x0000000000000000000000000000000000000000000000000000000000000001";
 export const DEFAULT_ENABLE_SIMULATION = false;
 export const DEFAULT_WAIT_FOR_TX_COMPLETION = false;
+export const DEFAULT_MAX_GAS_FOR_SUPRA_TRANSFER_WHEN_RECEIVER_EXISTS = 10;
+export const DEFAULT_MAX_GAS_FOR_SUPRA_TRANSFER_WHEN_RECEIVER_NOT_EXISTS = 10;
```
[the remark this answers is not in the document]
Comment (vpanchal-supra) [reply src/constants.ts:13]: Thanks, Just missed it

File: src/index.ts (hunk 1 above)
Comment (vpanchal-supra) [reply src/index.ts:1030]: This was a redundant check, as we are doing simulation before tx

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "supra-l1-sdk",
-  "version": "1.2.0",
+  "version": "1.3.0",
```
[the remark this answers is not in the document]
Comment (vpanchal-supra) [reply package.json]: Done

File: src/constants.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (vpanchal-supra) [reply src/constants.ts]: Done

**Review by isaacdoidge — APPROVED** [review]

**Pull request merged**